src/mixins/filing-template-mixin.ts

```diff
@@ -59,6 +59,7 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
   @Getter isEntityTypeSP!: boolean
   @Getter isEntityTypeGP!: boolean
   @Getter getCorrectedFilingType!: FilingTypes
+  @Getter getClientErrorCorrection!: string
```

Comment:
@severinbeauvais I think this getter should include both values "CLIENT/STAFF", because this getter will be used below for the `filing` variable.

Reply:
Are you saying that this getter should return an enum whose value is either CLIENT, STAFF, or null? If so then I agree.

Reply:
Yes. Maybe call it getCorrectionErrorType or something.